ivadomed/config/config_dataset.json

```diff
@@ -0,0 +1,191 @@
+{
```

Comment:
Would it be possible to update this config with a working "FILES" path?
I tried doing a "real-life" training from the CLI by following the structures in `example_loader_v2_configs.py` but I was unable to modify the config and make it work.
In particular, I'm not sure to understand what the different levels of `expected_input` and `expected_gt` stand for and I got this error:
```
    load_entry_point('ivadomed', 'console_scripts', 'ivadomed')()
  File "/home/mhbourget/code/ivadomed/ivadomed/main.py", line 898, in run_main
    run_command(context=context,
  File "/home/mhbourget/code/ivadomed/ivadomed/main.py", line 541, in run_command
    execute_multi_path_training(
  File "/home/mhbourget/code/ivadomed/ivadomed/main.py", line 693, in execute_multi_path_training
    all_data = AllDatasetGroups(
  File "/home/mhbourget/code/ivadomed/ivadomed/loader/all_dataset_group.py", line 45, in __init__
    self.n_expected_input: int = int(dict_all_datasets_group_spec.get(DataloaderKW.EXPECTED_INPUT, 0))
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```

Reply:
Good point. I think config_dataset.json was suppose to fill that role but I did not fully flush out that example so I will do that to make CLI testing more viable as well based on Mocker results instead of just from unit test.

Reply:
I have now updated the `config_dataset.json` with a fully flushed out example adapted from test_training.py. Note that you MAY have to search and replace/adjust the `"path_data": "C:\\git\\ivadomed\\tmp"` part as that is the default folder of mocker output for my running which will likely be different from yours.